template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-CSharp/.template.config/template.json

```diff
@@ -59,6 +59,10 @@
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
       "path": "UnitTest1.cs"
     },
+    {
+      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "MSTestSettings.cs"
+    },
```

Comment:
It's not clear to me whether this is needed of not. I have seen that the `Program.fs` for F# is not listed, I am not sure if that's a bug or expected.